ical_exporter.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import "io" 
+
+type ICalTaskExporter struct {
```

Comment:
I think we should think and discuss a bit more the design. We have `repositories` and `exporters` which may be a bit ambiguous. if we think very high level, we are logging time and outputting it somewhere, some questions are:

- do we want only one type of output / persistence ?
- do we want to consider an output as a way of persistence ? imagine we configure it to just save in ical, this means it should implement a `repository` interface not an `exporter` interface
- do we want to persist in several repositories ? if so we would need a registry. how are we thinking of this ?

These just came to my mind very quickly, more questions will probably come. maybe we should think a bit about how we want to look at our problem and design our code accordingly because currently it feels `repository` and `exporter` interface kind of do the same

Reply:
I didn't invest much time thinking about this yet. 
My logic here was that i didn't want to implement a parser for the ical file format. 

The current implementation of `RepositoryInterface` is more of a importer exporter where it knows how to read and write `Tasks` into some format. 
I predict this will change either to a CRUD like interface or more focused on the operations around tasks (stop task X, edit task activity start date, ...). 

I was seeing the app using a single repository only.